### PR TITLE
Adding issue template and code analysis github action

### PR DESCRIPTION
**Pull request by vibhansa-msft**
- Small changes in Issue template
- Github was suggesting an action of code analysis using codeql to get some security related scans so added that as well.

### CODE REVIEW

**Review by souravgupta-msft — APPROVED** [review]

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+### Which version of the blobfuse was used ?
```
Comment (gapra-msft) [inline .github/ISSUE_TEMPLATE.md:1]:
```suggestion
### Which version of blobfuse was used ?
```

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+### If relevant, please share your mount command.
+
+
+### What problem was encountered ?
+
+
+### Have you found a mitigation/solution ?
+
+
+### Kindly share logs if available.
```
Comment (gapra-msft) [inline .github/ISSUE_TEMPLATE.md:16]:
```suggestion
### Please share logs if available.
```

**Review by gapra-msft — APPROVED** [review]
Just have some small comments, otherwise looks good

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (tasherif-msft) [inline .github/ISSUE_TEMPLATE.md:1]:
```suggestion
### Which version of blobfuse was used?
```

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+### Which version of the blobfuse was used ?
+
+
+### Which OS (please include version) are you using ?
```
Comment (tasherif-msft) [inline .github/ISSUE_TEMPLATE.md:4]:
```suggestion
### Which OS distribution and version are you using?
```

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+
+### If relevant, please share your mount command.
+
+
+### What problem was encountered ?
```
Comment (tasherif-msft) [inline .github/ISSUE_TEMPLATE.md:10]:
```suggestion
### What was the issue encountered?
```

File: .github/ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+
+### Have you found a mitigation/solution ?
```
Comment (tasherif-msft) [inline .github/ISSUE_TEMPLATE.md:13]:
```suggestion
### Have you found a mitigation/solution?
```

**Review by tasherif-msft — APPROVED** [review]
approved but please consider changes first

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (vibhansa-msft) [reply .github/ISSUE_TEMPLATE.md:1]: corrected

Comment (vibhansa-msft) [reply .github/ISSUE_TEMPLATE.md]: corrected

**Pull request merged**